Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -289,6 +324,9 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
     Memory::Write_U32(0x900, _BufferOut);
     break;
 
+  // The following events don't seem to be possible due to the command length being only 6 bits
+  // I don't want to delete this without more context on why it's there and what it's meant to do
+  /*
   case EVENT_REGISTER:  // async
```

Comment:
Maybe looking at the libogc SD code and the WiiBrew article on the IOS SD resource manager could be useful:
http://wiibrew.org/wiki//dev/sdio/slot0